README.md

```diff
@@ -0,0 +1,65 @@
+# Letter Generator for datenanfragen .de
```

Comment:
Even though this project has been developed specifically for datenanfragen.de/datarequests.org, users are encouraged to use this for their own projects, so let's maybe keep the title more general and hint to datenanfragen in the text.